src/ios/Utilities.m

```diff
@@ -24,6 +24,10 @@ + (NSDate*)getApplicationBuildTime{
     return fileDate;
 }
 
++ (BOOL)cordova6OrGreater{
+    return NSClassFromString(@"CDVWebViewEngine");
```

Comment:
The function name is a bit confusing, because we actually check not for the `cordova 6`, but for existence of `CDVWebViewEngine` class, so if it get retired in e.g. cordova 7 it would be hard to identify just from code.

Can we please rename to something like `CDVWebViewEngineSupported` and cache the return value, so that subsequent calls were cheap?

Reply:
Thanks, done.